### PR TITLE
Add toasts after settings im-/export

### LINKED ISSUE #30: Feedback on settings export
There's no way to see if exporting the settings finished correctly. There should be a toast notification or something.

### PR DESCRIPTION
**Pull request by ghost**
Closes #30

### CODE REVIEW

File: src/settings_import_export.py [hunk 1]
```diff
@@ -39,19 +40,29 @@ def import_settings(path):
                     subprocess.run(['gsettings', 'set', \
                         'io.github.fsobolev.Cavalier', line.split(' ')[0], \
                         line.replace(line.split(' ')[0], '').strip()])
+        toast_msg = 'Settings sucessfully imported'
```
Comment (fsobolev) [inline src/settings_import_export.py]: Please use gettext for all toast messages strings, put them inside `_()` (e.g. `_('Settings sucessfully imported)`)

File: src/settings_import_export.py [hunk 2]
```diff
@@ -39,19 +40,29 @@ def import_settings(path):
...
+
     except Exception as e:
         print('Can\'t import settings from file: ' + path)
         print(e)
+        toast_msg = 'Failed to import settings'
+
+    Adw.PreferencesWindow.add_toast(window, Adw.Toast.new(toast_msg))
```
Comment (fsobolev) [inline src/settings_import_export.py]: you can shorten this to `window.add_toast(Adw.Toast.new(toast_msg))`, same on line 68

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: src/settings_import_export.py [hunk 3]
```diff
@@ -58,11 +58,11 @@ def export_settings(window, path):
...
+        toast_msg = _('File successfully saved')
 
     except Exception as e:
         print('Can\'t export settings to file: ' + path)
         print(e)
-        toast_msg = 'Failed to save file'
+        toast_msg = _('Failed to save file')
 
-    Adw.PreferencesWindow.add_toast(window, Adw.Toast.new(toast_msg))
+    window.add_toast(window, Adw.Toast.new(toast_msg))
```
Comment (fsobolev) [inline src/settings_import_export.py]: do not pass window as argument
`window.add_toast(Adw.Toast.new(toast_msg))`

**Comment by ghost** [comment]
@fsobolev Was that all?

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: src/settings_import_export.py (hunk 3 above)
Comment (fsobolev) [reply src/settings_import_export.py]: when you used `Adw.PreferencesWindow.add_toast(window, Adw.Toast.new(toast_msg))`, you called a class method that required an instance as the first argument, but when the method is called on the instance itself, you don't need to pass it as an argument: `window.add_toast(window, Adw.Toast.new(toast_msg))` = `Adw.PreferencesWindow.add_toast(window, window, Adw.Toast.new(toast_msg))`, this causes an error:
```
TypeError: Adw.PreferencesWindow.add_toast() takes exactly 2 arguments (3 given)
```

Comment (ghost) [reply src/settings_import_export.py]: That makes sense. I'll correct it right away.

**Review by fsobolev — APPROVED** [review]
